tapir/shifts/models.py

```diff
@@ -721,6 +721,35 @@ def get_current_shift_exemption(self, date=None):
     def is_currently_exempted_from_shifts(self, date=None):
         return self.get_current_shift_exemption(date) is not None
 
+    def send_shift_reminder_emails(self):
+        today = datetime.date.today()
+        next_monday = today + datetime.timedelta((0 - today.weekday()) % 7)
```

Comment:
maybe make `days=` explicit here? Had to look up if it really works (it does :P)

Reply:
Haha it shows that I copy-pasted that bit, I didn't notice the days= was missing but would also have guessed that it doesn't work!